config/environments/production.rb

```diff
@@ -23,19 +25,18 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = 'http://assets.example.com'
-
-  # Compress CSS using a preprocessor.
-  # config.assets.css_compressor = :sass
-
-  # Fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = true
```

Comment:
not sure we should remove this. In Rails code I see

```erb
  <%- unless options.skip_sprockets? -%>
  # Compress CSS using a preprocessor.
  # config.assets.css_compressor = :sass

  # Do not fallback to assets pipeline if a precompiled asset is missed.
  config.assets.compile = false

  <%- end -%>
```

and I think we are using `sprockets` for the admin site.

Reply:
Returned back this option.